week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -14,8 +14,33 @@
  * </div>
  */
 function exerciseOne(arrayOfPeople) {
-  let content = document.querySelector("#content");
+  namesList();
+  jobsList();
+
+  function namesList(){
```

Comment:
I think these functions should be outside of the `exerciseOne` function. Could you move them outside?